src/services/web3/web3.plugin.ts

```diff
@@ -122,6 +125,21 @@ export default {
           lsSet('connectedWallet', account.value);
           lsSet('connectedProvider', wallet);
           pluginState.walletState = 'connected';
+
+          // special case for polygon
+          if (
+            configService.env.NETWORK === '137' &&
```

Comment:
same configs constant comment as above. 
also im having second thoughts about pulling this if statement logic out of the connect wallet function, thoughts?